package.json

```diff
@@ -56,9 +56,11 @@
     "start:components-angular": "yarn --cwd ./packages/components-angular start",
     "start:components-react": "yarn --cwd ./packages/components-react start",
     "start:components-vue": "yarn --cwd ./packages/components-vue start",
+    "start:crawler": "yarn --cwd ./packages/crawler start",
     "start:fallbacks": "yarn --cwd ./packages/fallbacks start",
     "start:storefront": "yarn --cwd ./packages/storefront start",
     "start:utilities-deprecated": "yarn --cwd ./packages/utilities-deprecated start",
+    "start:assets": "yarn --cwd ./packages/assets start",
```

Comment:
Let's revert this since it doesn't make a lot of sense to start the cdn like this.

Reply:
true, fixed